### ISSUE TITLE
pandoc revealjs command

### ISSUE THREAD

**Issue by koenverbruggen** [janboone/open_source_for_economists#7]
Hi,
To convert the markdown file into the html presentation, I ran the following code: pandoc -s --mathjax --slide-level 2 --toc --toc-depth=1 -t revealjs presentation.md -V theme=solarized -o presentation.html

This gives no error, but the result is not the same sort of presentation, actually it is just a plain html file, altough mathjax works, but I guess something is wrong with revealjs.

What I did:

I cloned the entire repository: https://github.com/janboone/markdown_presentation_files.. then used the above stated code, with changing presentation in presentation1.. the result should in my view be the same as presentation.html, but it is not, looks more like a plain html file..any ideas on what is going wrong ?

Kind Regards,

Koen

**Comment by janboone** [comment]
Indeed, there are a couple of students with this problem. I don't quite understand how it happens, but I know how to solve it (thanks to @bartdegeus):

Go to this website:

https://github.com/hakimel/reveal.js/releases/tag/3.2.0

and download (at the bottom of the page: source code zip).

Unzip this file into the folder reveal.js-3.2.0. 

Rename the folder to reveal.js

Copy the folder in your directory with your presentation and replace the existing folder reveal.js in that directory.

Then your presentation should work.

Hope that this helps.

**Comment by koenverbruggen** [comment]
Thanks, this worked!
